docs/contracts/universal-router2/02-technical-reference.md

```diff
@@ -0,0 +1,355 @@
+---
+id: technical-reference
+title: Technical Reference
+sidebar_position: 1
+---
+
+## Functions
+
+Transactions to the `UniversalRouter` all go through the `execute` function:
+
+- `execute(bytes calldata commands, bytes[] calldata inputs, uint256 deadline)`
+- `execute(bytes calldata commands, bytes[] calldata inputs)`
+
+The first overload allows for a transaction `deadline`. If the current `block.timestamp` is beyond this deadline, the transaction will revert. Otherwise, both versions function identically.
```

Comment:
this line is confusing to me, especially if a developer mistakens that `execute(bytes calldata commands, bytes[] calldata inputs)` overloads the function with the deadline

---

I would just explain the functions as-is. Something like: "Both functions behave and process the commands the exactly the same, the first one with an deadline. The function without the deadline parameter will not revert based on block.timestamp"